Xamarin.Essentials/Email/Email.ios.cs

```diff
@@ -1,14 +1,35 @@
 ﻿using System.Threading.Tasks;
+using Foundation;
 using MessageUI;
+using UIKit;
 
 namespace Xamarin.Essentials
 {
     public static partial class Email
     {
         internal static bool IsComposeSupported
-            => MFMailComposeViewController.CanSendMail;
+        {
+            get
+            {
+                var can = MFMailComposeViewController.CanSendMail;
+                if (!can)
+                {
+                    var url = NSUrl.FromString("mailto:");
+                    NSRunLoop.Main.InvokeOnMainThread(() => can = UIApplication.SharedApplication.CanOpenUrl(url));
+                }
+                return can;
+            }
+        }
```

Comment:
First check `MFMailComposeViewController` and then try `mailto:`.

This is a bit stupid since there is always a mail app, and at the same time the only app supported by mailto is the default app - even if Outlook is installed and set up, the mailto will keep asking you to set up an account in the Apple app.